packages/react-api/src/hooks/useCartoLayerProps.js

```diff
@@ -6,9 +6,14 @@ import { MAP_TYPES, API_VERSIONS } from '@deck.gl/carto';
 export default function useCartoLayerProps({
   source,
   uniqueIdProperty,
-  viewportFeatures = true
+  viewportFeatures = true,
+  debounceTimeOut = 500
```

Comment:
We should probably be a bit more explicit on this new public parameter in useCartoLayerProps (it looks like a nice name for useViewportFeatures though), to indicate where it applies. How about viewporFeaturesDebounceTimeout ?

Reply:
Totally agree. I'll change it.